### PR TITLE
add delay command line parameter

### PR DESCRIPTION
**Pull request by mbbush**
This just ports over the --delay command line parameter from master to alpha, and passes it all the way down the call chain.

### CODE REVIEW

File: stores/amazon_handler.py [hunk 1]
```diff
@@ -98,6 +100,7 @@ async def run_async(self, checkout_tasks=1):
             item_list=self.item_list,
             amazon_config=amazon_config,
             tasks=checkout_tasks,
+            delay=delay,
```
Comment (DakkJaniels) [inline stores/amazon_handler.py:103]: `delay=self.delay`?

Comment (mbbush) [reply stores/amazon_handler.py:103]: Good catch, thanks.

**Comment by mbbush** [comment]
Fixed silly mistakes, ran black, tested it and it works on my machine (which wasn't true when I first opened the PR).

**Review by DakkJaniels — APPROVED** [review]
LGTM, thanks!

**Pull request merged**